introduction.md

```diff
@@ -1,4 +1,8 @@
 Hi! This is Smriti!
 Creating my first clone of project.
 Favorite codeschool path JavaScript
+List of deadly skills
+* Git
+* JavaScript
+* Ruby
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?